pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[project]
+name = "rocketpy"
+version = "1.2.2"
+description="Advanced 6-DOF trajectory simulation for High-Power Rocketry."
+dynamic = ["dependencies", "optional-dependencies"]
+readme = "README.md"
+requires-python = ">=3.8"
+authors = [
+  {name = "Giovani Hidalgo Ceotto", email = "ghceotto@gmail.com"}
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Topic :: Rocket Flight Simulation :: Libraries :: Python Modules",
+]
+
+[project.urls]
+homepage = "https://rocketpy.org/"
+documentation = "https://docs.rocketpy.org/"
+repository = "https://github.com/RocketPy-Team/RocketPy"
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+py-modules = []
+
+[tool.setuptools.package-data]
+drag_curves = ["**/*.csv"]
+engine_files = ["**/*.eng"]
+environment_files = ["**/*.nc"]
+jupyter_notebooks = ["**/*.ipynb"]
+documentation = ["**/*.rst"]
+
+[tool.setuptools.dynamic]
+dependencies = {file = ["requirements.txt"]}
+optional-dependencies.env_analysis = {file = ["requirements-optional.txt"]}
+optional-dependencies.test = {file = ["requirements-tests.txt"]}
+
+[tool.black]
+line-length = 88 
+include = '\.py$'
```

Comment:
Could we also include .ipynb to be formatted?

```
include = '\.py$'
```

Reply:
sure